packages/bpytop-git/bpytop-git.pacscript

```diff
@@ -21,5 +21,12 @@ build() {
 }
 
 install() {
-    sudo make install DESTDIR=/usr/src/pacstall/bpytop-git
-} 
+    sudo make install DESTDIR=/usr/src/pacstall/$name
+    sudo install -Dm644 LICENSE /usr/src/pacstall/$name/usr/share/licenses/$pkgname
+    sudo install -Dm644 README.md /usr/src/pacstall/$name/usr/share/doc/$pkgname
```

Comment:
```suggestion
    sudo install -Dm644 LICENSE -t /usr/src/pacstall/$name/usr/share/licenses/$pkgname
    sudo install -Dm644 README.md -t /usr/src/pacstall/$name/usr/share/doc/$pkgname
```

These need to be inside the `pkgname` folder, not their respective parent folder(`license`/`doc`)

Currently it's like this:
![img](https://0x0.st/-9Te.png)